test/images/network-tester/Dockerfile

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 
 FROM bash
+RUN apk add --update \
```

Comment:
Minor: `RUN apk add --no-cache curl` instead.
> As of Alpine Linux 3.3 there exists a new `--no-cache` option for `apk`. It allows users to install packages with an index that is updated and used on-the-fly and not cached locally: [...]
> This avoids the need to use `--update` and remove `/var/cache/apk/*` when done installing packages.

This should work given the `bash` image is on Alpine 3.6:
https://github.com/tianon/docker-bash/blob/75358344e900aa56e64722b2241a6bcd39ca67ee/4.4/Dockerfile#L1